src/components/HomePage/AccountCard/AccountCard.tsx

```diff
@@ -34,7 +34,7 @@ const AccountCard = ({
         <div className="description">
           <div> {t('account_card.tx_latests')} </div>
           <div>
-            <div className="see-all" onClick={onSeeAll}>
+            <div className="see-all" onClick={() => onSeeAll(transactions)}>
```

Comment:
Can we avoid using lambdas here and move it to a function? Not sure how are you handling it using hooks (CC @nicosantangelo). I'm imaging something like:

```ts
const handleOnSeeAll = () => props.onSeeAll(transactions)

// ....

return (
// ...

<div className="see-all" onClick={handleOnSeeAll}>
```

Reply:
Done! but I don't know if it is a performance improvement besides the readability.